### ISSUE TITLE
Problems running ChocolateyGUI on Windows 10 Technical Preview

### ISSUE THREAD

**Issue by sevenday4** [chocolatey/ChocolateyGUI#183]
Sorry Gary I tried earlier to put the snapshot here, but  the attach image selections is greyed out. Try looking at this snapshot using this URL: https://onedrive.live.com/redir?page=view&resid=7D6A6736229274CE!43783&authkey=!ADN9ipJin7HqOf8 And yes, that was me, I am sevenday4.

**Comment by RichiCoder1** [comment]
I did end up running ChocoGUI with no issues, but I'm going to say we say we don't start work on official support until post 0.12.

**Comment by gep13** [comment]
Can I get anyone who is following this thread to try out a new version of ChocolateyGUI using the following command:

`choco install chocolateygui -version 0.13
.0-PullRequest0243 -source https://ci.appveyor.com/nuget/chocolateygui-36tajo4x9yei -pre`

This is a new release of ChocolateyGUI which includes the ability to use the latest Chocolatey, i.e. 0.9.9.x, and it should hopefully solve a lot of the problems that have been reported lately.

Please give it a try, and let me know if you are still running into issues.

**Comment by johnnyramos** [comment]
Is that version the same code as the one on the develop branch?

**Comment by gep13** [comment]
@johnnyramos no, that build is from a Pull Request, that hasn't yet been merged into develop

**Comment by johnnyramos** [comment]
Chocolatey v0.9.9.2
Installing the following packages:
chocolateygui
By installing you accept licenses for the packages.
chocolateygui not installed. An error occurred during installation:
 Unable to resolve dependency 'PowerShell'.
chocolateygui install not successful.
chocolateygui not installed. An error occurred during installation:
 Unable to resolve dependency 'PowerShell'.

Chocolatey installed 0/1 package(s). 1 package(s) failed.
 See the log for details.
Failures:
 - chocolateygui

**Comment by gep13** [comment]
yeah, if you don't already have PowerShell installed, you will need a slightly different command to pull in the pre-requistes, as the AppVeyor feed doesn't know about upstream source.

Try:

`choco install chocolateygui -version 0.13.0-PullRequest0243 -source https://ci.appveyor.com/nuget/chocolateygui-36tajo4x9yei;http://chocolatey.org/api/v2/ -pre

**Comment by johnnyramos** [comment]
Weird, I mean - I'm on Windows 8.1 and work off powershell alot. Is there anyway to skip the req?

Name                           Value
----                           -----
PSVersion                      4.0
WSManStackVersion              3.0
SerializationVersion           1.1.0.1
CLRVersion                     4.0.30319.0
BuildVersion                   6.3.9600.17400
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0}
PSRemotingProtocolVersion      2.2

**Comment by gep13** [comment]
Yes, but I have just stepped away from computer. Check the choco wiki install command for argument to ignore dependencies. The installation of the dep in this case wouldn't change anything, as it would see the PS v4 is already installed, and it would do nothing but tell Chocolatey that it is already installed.

Sent from my Windows Phone
________________________________

**Comment by johnnyramos** [comment]
Looks to be running well in Windows 8.1 and sees the packages for 0.9.9.2. :+1: 

Would test on Windows 10, but that has it's own OS bugs and won't allow me to run a cmd window.

**Comment by RichiCoder1** [comment]
It breaks league of legends for me.
#priorities

**Comment by gep13** [comment]
Breaks how?

Sent from my Windows Phone
________________________________

**Comment by RichiCoder1** [comment]
Windows 10 does, I should clarify, and it cause LoL to crash anytime I start a game. Unrelated issue, but it's the reason why I reverted my home machine to 8.1

**Comment by gep13** [comment]
ha ha, gotcha :+1:

**Comment by gep13** [comment]
I have just proved that on a Windows 7 VM, with Chocolatey 0.9.8.33, ChocolateyGUI functions as expected :-) Boom!

**Comment by ScottIsAFool** [comment]
I'm having an issue on 10041 of Win10. I have ChocolateyGUI installed, and it runs just fine, I can search, look at packages etc, however, when I click on install, the UI fades out (overlay), then waits a few seconds, then goes back to normal. Nothing further happens. No package is installed. Nothing. I've tried running it as administrator, with no joy either. Shame, I hate CLI :)

**Comment by gep13** [comment]
I have just set up a new VM with Windows 10 (10041), installed Chocolatey 0.9.9.2 and ChocolateyGUI 0.13.0 and I can confirm that it is all working as expected.  I am able to install and uninstall packages through ChocolateyGUI.

As per our Twitter discussion, it looks like you still had Chocolatey 0.9.8.33 installed (this is currently the default version that you will get when installing from the command on the website).  To get the latest version of Chocolatey installed, run:

```choco update chocolatey```

and once that is installed, run:

```choco --version```

to confirm that it is installed

![image](https://cloud.githubusercontent.com/assets/1271146/6886977/0ac8b4a8-d64a-11e4-97a0-9ea691480665.png)

With this test completed, I am happy to close this issue, as ChocolateyGUI now has been tested on Windows 10.  If anyone runs into any other issues, please raise a separate issue.

**Comment by drfrpjr** [comment]
I have been using Choco and ChocoGUI -- the latest versions -- with Windows 10 on a Surface. Works fine. Powershell version=5.0.10240.17443

However, I just got a new desktop PC and while Choco is fine, ChocoGUI viciously crashes with the error below. This is running as administrator, etc. I think I have tried everything else above as well. Powershell version=5.1.14393.953

Any advice?

Thanks. 

![image](https://user-images.githubusercontent.com/14821996/28343669-720031b6-6bec-11e7-9bb6-4b01457d5cc8.png)
